src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
```

Comment:
The body element is centered using flexbox, which is fine for a single-page layout. However, ensure that this styling doesn't interfere with other elements on the page if the layout becomes more complex.